proto/remote_execution.proto

```diff
@@ -2239,6 +2243,24 @@ message UsageStats {
 
   // IO PSI metrics.
   PSI io_pressure = 7;
+
+  message Timeseries {
+    // Time when recording started.
+    google.protobuf.Timestamp start = 1;
+
+    // Time when the last sample was taken.
+    google.protobuf.Timestamp end = 2;
+
+    // Duration between each sample.
+    google.protobuf.Duration period = 3;
+    
+    // Observed samples. The interpretation of these samples may depend on the
+    // metric being reported.
+    //
+    // TODO: maybe do quantization + delta-compression and/or RLE to reduce data
+    // size.
```

Comment:
I think if we do this we should just add a bytes field and link to a util function where we use roaring or something.

Reply:
OK - will keep this in mind